tests/end_to_end/candid_rpc/class_syntax/notify_raw/benchmarks_canister2.md

```diff
@@ -0,0 +1,18 @@
+## Current benchmarks Azle version: 0.25.0
+
+| Execution | Method Name         | Instructions | Cycles  | USD           |
+| --------- | ------------------- | ------------ | ------- | ------------- |
+| 0         | receiveNotification | 908,426      | 953,370 | $0.0000012743 |
```

Comment:
We should consider adding info on calling the function 1 million times or something, to give people some kind of real-world feeling for how much it will cost

Reply:
I say that mostly in jest but also I think per thousand calls is better. It keeps the price in fractions of pennies for most of them and I think a thousand is easier to visualize than a million

Reply:
lets do a million

Reply:
done